build.assets/versions.mk

```diff
@@ -0,0 +1,24 @@
+# Keep all tool versions in one place.
+# This file can be included in other Makefiles to avoid duplication.
+
+
+# Sync with devbox.json.
+GOLANG_VERSION ?= go1.20.6
```

Comment:
Please rebase, your Go and Buf versions are out of date (just off of my head).

Reply:
Aware. This PR is opened for a while and I don't want to keep updating it every time we bump a tool version.

Reply:
Yep, I noticed the dates later.